build/postbuild.ps1

```diff
@@ -59,8 +59,6 @@ foreach($nuspec in $nuspecs)
 				$item.Pushed = $true;
 			}
 		}
-		
-		$item
```

Comment:
I had this in there so you could see the status of whether it was pushed or not. Is there a reason you removed it?